lib/discover/hyperbahn.js

```diff
@@ -0,0 +1,75 @@
+var TChannelAsThrift = require('tchannel/as/thrift');
+var TChannel = require('tchannel');
+var fs = require('fs');
+var path = require('path');
+
+module.exports.discover = function hyperbahnDiscover(urlObj, original, callback) {
+    var host;
+    if (urlObj.host) {
+        host = urlObj.host;
+    } else {
+        host = '127.0.0.1:21300';
```

Comment:
It is the ip:port combination hyperbahn uses in their examples: https://github.com/uber/hyperbahn#getting-your-app-on-hyperbahn